opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java

```diff
@@ -356,6 +360,29 @@ void extract_onlyBaggage() {
     assertThat(spanContextShim.getBaggage()).isEqualTo(baggage);
   }
 
+  @Test
+  void extract_onlySampledTraceFlags() {
+    Map<String, String> map = new HashMap<>();
+    io.opentelemetry.api.trace.SpanContext invalidSpanContextWithFlags =
+        io.opentelemetry.api.trace.SpanContext.create(
+            TraceId.getInvalid(),
+            SpanId.getInvalid(),
+            TraceFlags.getSampled(),
+            TraceState.getDefault());
+    Context context =
+        Context.root().with(PropagatedOpenTelemetrySpan.create(invalidSpanContextWithFlags));
+    CustomTextMapPropagator textMapPropagator = new CustomTextMapPropagator();
+    textMapPropagator.setExtractReturnContext(context);
```

Comment:
This can never be done in a real propagator, which should be stateless. I would recommend changing CustomTextMapPropagator to look for debug header in the carrier and _then_ constructing `invalidSpanContextWithFlags` in response. That would constitute a realistic test.

Reply:
Thanks for the suggestion! I will make the change.

Reply:
@carlosalberto Updated. This is done.